docs/LDAP.rst

```diff
@@ -1057,3 +1057,47 @@ To resolve this, add an ACL (Access Control List) rule in the ``slapd.conf`` fil
 .. |image0| image:: images/LDAP/ldap_integration.png   
 .. |image1| image:: images/LDAP/ldap_user.png    
 .. |image2| image:: images/LDAP/GroupsDNs.png   
+
+=================================================
+Organization externalStore jobs configuration
+=================================================
+
+Organization import and synchronization is performed by IDMExternalStoreImportService.
```

Comment:
We should at first explain what is the goal and the principle of this import.

Reply:
goal and  principle of the import were added.